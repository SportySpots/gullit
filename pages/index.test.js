import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

test('Index renders & matches snapshot', () => {
  const component = renderer.create(
    <Home />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
