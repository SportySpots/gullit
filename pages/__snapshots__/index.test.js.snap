// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders & matches snapshot 1`] = `
<div>
  <a
    href="/games"
    onClick={[Function]}
  >
    games page
  </a>
  Welcome to Next.js!
</div>
`;
